test/functional/apps/visualize/_tile_map.ts

```diff
@@ -125,26 +125,26 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
         it('Fit data bounds should zoom to level 3', async function () {
```

Comment:
I verified this change. The test seems to behave correctly. The map correctly fits to the data bounds.